api/host.go

```diff
@@ -129,6 +129,9 @@ type (
 	HostInfo struct {
 		PublicKey  types.PublicKey `json:"publicKey"`
 		SiamuxAddr string          `json:"siamuxAddr"`
+
+		Prices   HostPriceTable     `json:"prices"`
+		Settings rhpv2.HostSettings `json:"settings"`
```

Comment:
We don't want the `Prices` and `Settings` here. The bus doesn't keep valid price tables. That will be the responsibility of the client. The client only trusts that the hosts are good according to the bus but fetches prices itself.

Reply:
Hm ok. I was hesitant to add it of course. I thought about return `HostInfo` and some other internal type from the store, like `HostGouging` or something... Definitely wanted to avoid querying it separately, so.. instead I pass a gouging checker to the store to filter the hosts.

Isn't your statement a little contradicting? We want to return usable hosts, but the `bus` can't be aware of prices. I guess the endpoint could accept host filters of sorts (including gouging).